perceval/utils/algorithms/circuit_optimizer.py

```diff
@@ -149,6 +149,7 @@ def _gen_ps(i: int):
             template_component_generator_func,
             phase_shifter_fun_gen=_gen_ps,
             phase_at_output=phase_at_output)
+        barrier_free_template = Circuit(template.m)
```

Comment:
I do not see how `barrier_free_template`  is used :-(

Reply:
Oups, I need to delete that, my bad